### PR TITLE
Add KVM configuration instructions and example.

### PR DESCRIPTION
**Pull request by welteki**
## Description

Add KVM configuration instructions and example

## Motivation and Context

Document how to enable support for KVM.

## How Has This Been Tested?

Verified site and instruction locally.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

### CODE REVIEW

File: docs/examples/kvm-guest.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            fetch-depth: 1
+        - name: Install arkade
+            uses: alexellis/setup-arkade@v2
+        - name: Install firecracker
+            run: sudo arkade system install firecracker
+        - name: Run microVM
+            run: sudo ./run-vm.sh
+    ```
+
+2. Add the `run-vm.sh` script to the root of the repository.
```
Comment (alexellis) [inline docs/examples/kvm-guest.md:62]: Could you make the build clone your repo instead?

Then just keep the bullets of what it does?

**Review by alexellis — CHANGES_REQUESTED** [review]
Quick change please

File: docs/examples/kvm-guest.md (hunk 1 above)
Comment (welteki) [reply docs/examples/kvm-guest.md:62]: Updated the script to clone the example repo and tun the VM script from that.

**Review by alexellis — APPROVED** [review]
Approved

**Pull request merged**